src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/ProxyAcceptStream.java

```diff
@@ -136,7 +136,7 @@ private void handleBegin(BeginFW begin)
             // Should already be canonicalized in http / http2 nuklei
             final String requestURL = getRequestURL(requestHeaders);
 
-            this.requestURLHash = requestURL.hashCode();
+            this.requestURLHash = requestURL.hashCode() + authorizationScope;
```

Comment:
The usual way to combine to hash codes is 31 * code1 + code2 (see List.hashCode and Arrays.hashCode(int[]), so I would recommend following that technique